addons/pinniped/config-controller/manifests/role.yaml

```diff
@@ -18,6 +18,7 @@ rules:
   resources:
   - secrets
   verbs:
+  - delete
```

Comment:
remind me where we use this one? i thought we were going to rely on owner references to ensure that secrets get deleted

Reply:
I kept that around for future deletion in `v1alpha1` scenarios, but I can remove it for now since we don't need it until the delete story.